feature/google/src/latestDependencies/java/com/revenuecat/purchases/google/billingClientParamBuilders.kt

```diff
@@ -41,3 +50,40 @@ val SubscriptionOfferDetails.subscriptionBillingPeriod: String?
 
 val SubscriptionOfferDetails.isBasePlan: Boolean
     get() = this.pricingPhases.pricingPhaseList.size == 1
+
+fun PurchaseOption.buildPurchaseParams(storeProduct: StoreProduct): BillingFlowParams? {
```

Comment:
Hmm I wonder if we should keep this inside the Billing wrapper... I don't think it will be used outside of that right?

Reply:
i thought we kinda had a norm to put extension functions in separate files, but i think if we only use it in BillingWrapper that's a stronger argument..will move

Reply:
Yeah... Also, not sure if we want to make it an extension function. I'm ok either way, but this piece of logic seems to belong only to the purchase flow so I would keep it there